components/actionbar/index.css

```diff
@@ -147,27 +113,15 @@
 
 	/* action group */
 	.spectrum-ActionGroup {
-		margin-inline-end: var(--mod-actionbar-spacing-action-group-end, var(--spectrum-actionbar-spacing-action-group-end));
-		margin-block-start: var(--mod-actionbar-spacing-action-group-top, var(--spectrum-actionbar-spacing-action-group-top));
-
 		/* align to end by default */
 		margin-inline-start: auto;
+		padding-inline-start: var(--mod-actionbar-spacing-label-to-actiongroup, var(--spectrum-actionbar-spacing-label-to-action-group)); /* space between label and action group */
```

Comment:
After the property renaming, the mod needs to be renamed as well. The changeset also still references "label-to-actiongroup".